packages/@jsii/java-runtime/project/src/main/java/software/amazon/jsii/JSException.java

```diff
@@ -0,0 +1,32 @@
+package software.amazon.jsii;
+
+/*
+ * Represents an error thrown from the user JS library.
+ */
+public final class JSException extends JsiiBaseException {
```

Comment:
this will be inherited from the base class though, no?

Reply:
Statics are not inherited nor inheritable. Java allows you to refer to a static declared on a parent type though a child type, but this is actually not inheritance (and best practices encourage you to avoid doing that, precisely because the semantics might change if the child class adds a new declaration, shadowing -- and not overriding -- the one from the parent class).